README.md

```diff
@@ -1,50 +1,120 @@
 # Module controlled-components 
 
-Provide a description of the purpose of the module and any relevant information.
+This module houses components that utilize sensor feedback to transform sensor readings and/or control actuators.
 
 ## Model viam:controlled-components:sensor-controlled
 
-Provide a description of the model and any relevant information.
+The `sensor-controlled` model is a base that combines movement sensors with PID controls to actuate a `base` component.
 
 ### Configuration
 The following attribute template can be used to configure this model:
 
 ```json
 {
-"attribute_1": <float>,
-"attribute_2": <string>
+"base": <string>,
+"movement_sensor": [<string>, <string>],
+"control_frequency_hz": <float>,
+"control_parameters": [
+    {
+        "type": "linear_velocity",
+        "p": <float>,
+        "i": <float>,
+        "d": <float>,
+    },
+    {
+        "type": "angular_velocity",
+        "p": <float>,
+        "i": <float>,
+        "d": <float>
+    },
+  ]
```

Comment:
seeing this config again made me really want to change this config again. I don't like that the user has to specify `type`, as the config structure should handle that.

Reply:
Do you want it to be flatter?
ie 
```
angular_p: <>
....
linear_p: <>
....
```

Reply:
something like 
```
linear_pid: {
p: <>
i: <>
d: <>
}
angular_pid: {
p: <>
i: <>
d: <>
}
````
this way its clear each set of pids are grouped together, while removing the type thing.

can maybe make a quick scope for it once scb gets removed from rdk